packages/suite/src/components/suite/modals/RequestEnableTor.tsx

```diff
@@ -72,8 +79,18 @@ export const RequestEnableTor = ({ onCancel, decision }: RequestEnableTorProps)
                         <Button variant="secondary" onClick={onCancel}>
                             <Translation id="TR_TOR_REQUEST_ENABLE_FOR_COIN_JOIN_LEAVE" />
                         </Button>
-                        <Button variant="primary" onClick={onEnableTor}>
-                            <Translation id="TR_TOR_ENABLE" />
+
+                        <Button
+                            variant="primary"
+                            onClick={onEnableTor}
+                            isLoading={isTorLoading}
+                            isDisabled={isTorLoading}
```

Comment:
nit: setting `isDisabled` on the same value as `isLoading` is kind of redundant, since Button is disabled when isLoading is truthy